cron/notification-for-bekrachtigde-mandataris.ts

```diff
@@ -76,9 +78,10 @@ async function HandleEffectieveMandatarissen() {
 }
 
 async function fetchEffectiveMandatarissenWithoutBesluit() {
-  const tenDaysBefore = new Date();
-  tenDaysBefore.setDate(tenDaysBefore.getDate() - 10);
-  const escapedTenDaysBefore = sparqlEscapeDateTime(tenDaysBefore);
+  const momentTenDaysAgo = moment(new Date()).subtract(10, 'days');
+  const escapedTenDaysBefore = sparqlEscapeDateTime(
+    new Date(momentTenDaysAgo.format('YYYY-MM-DD')),
```

Comment:
use momentTenDaysAgo.toDate() this is scary because of us vs eu order

Reply:
Updated to toDate() bebugger value looks good!